src/Utils.cpp

```diff
@@ -18,13 +18,15 @@
 #include <QXmlStreamReader>
 
 #include <cmath>
+#include <qtextboundaryfinder.h>
```

Comment:
You might want to use <QTextBoundaryFinder> instead? Just a little nitpick.

Reply:
```suggestion
#include <QTextBoundaryFinder>
```